src/drafts/MarkdownEditor/_useListEditing.ts

```diff
@@ -45,7 +45,7 @@ const isNumericListItem = (item: ListItem | null): item is NumericListItem => ty
 export const parseListItem = (line: string): ListItem | null => {
   const result = listItemRegex.exec(line)
   if (!result) return null
-  const [, leadingWhitespace = '', fullDelimeter, itemNumberStr = '', taskBox = null, text] = result
+  const [, leadingWhitespace = '', fullDelimeter, itemNumberStr = '', , taskBox = null, text] = result
```

Comment:
Here you are skipping the capture group for the one or two spaces, meaning that the updated Markdown will always have only one space. Do we maybe want to preserve the number of spaces like we do with `leadingWhitespace` here?

Reply:
Addressed in [ad9c47f](https://github.com/primer/react/pull/3972/commits/ad9c47fc25f14fd1c5456de51560c7766b883512)